CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 
 * Bump: log4j to 2.16.0 (#1845)
+* Bump: logback to 1.2.9
```

Comment:
@jcaraballo merge `main` into your branch, it's not up to date.

Reply:
I've rebased. Thank you for pointing it out.

Reply:
@jcaraballo no worries, its still wrong, this entry should be below the `Unreleased` tag